domains/deivid.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "deividcomsono",
+        "email": "",
+        "discord": "773212482835710032"
+    },
+    "record": {
+        "CNAME": "deividcomsono.github.io"
```

Comment:
Simple Hello World at the moment, would ask for some more site content before I release this domain to you.